### ISSUE TITLE
Zooming with CTRL+wheel is reported to be unstable

### ISSUE THREAD

**Issue by stephanrauh** [stephanrauh/ngx-extended-pdf-viewer#2465]
[2024-07-16 12-02-11.webm](https://github.com/user-attachments/assets/051d8c93-9387-429e-9bd5-44fbc7d65af3)
Look at recording because it's hard to reproduce. It started not from the previous version but some of previous. When changing the zoom, sometimes it hardstack in some places(ctrl + wheel). After changing zoom it helps sometimes in case of scroll appear. Could help understand what is this problem, how to solve it? Output method like (currentZoomFactor) (zoomChange) stop work in this case(stop emmit a value). On your demo I can reproduce only for a few times and scroll enough for unblock it. Thanks in advance.

_Originally posted by @YaromichSergey in https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/2337#issuecomment-2230517285_

**Comment by YaromichSergey** [comment]
Hello, any news about this problem?

**Comment by stephanrauh** [comment]
Not yes. Thanks for nudging me - it helps me to address the important (or most-needed) tickets first.

**Comment by stephanrauh** [comment]
I believe to have solved this ticket, for example by implementing #3069. There's a follow-up ticket, #3171, because I'm still not happy - but not it's good enough to be useful.
